src/components/HideableText/HideableText.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import {usePrivacyModeContext} from '../../Settings/PrivacyMode/PrivacyModeContext'
+import {Text} from '..'
+
+type Props = {
+  children: string
+}
+
+export const HideableText = ({children}: Props) => {
+  const {privaceModeStatus} = usePrivacyModeContext()
+  return <Text>{privaceModeStatus === 'SHOWN' ? children : children.replaceAll(/./g, '*')}</Text>
```

Comment:
The fact of hiding all the characters or ignoring the spaces and line breaks, is something that will have to be consulted with the product/design, right?